### DISCUSSION TITLE
404 Error

### DISCUSSION THREAD

**Discussion by PeterGyger** [community/community#21697] [Other Feature Feedback, Questions, & Ideas]
<p>Hello</p>
<p>Yesterday i change the name from my Repository from “<a href="http://petergyger.github.io" rel="nofollow noopener">petergyger.github.io</a>” to a other name.</p>
<p>Then i create a new Repository (Test Use) and use this Domain Name.</p>
<p>After the Test i changend back. But now i habe a Error:  “404 There isn’t a GitHub Pages site here”.</p>
<p>What can be the reason? What can i check?</p>
<p>Thanks a lot for every hint.</p>
<p>Greetings</p>
<p>Peter </p>

**Comment by Wabri** [comment, marked as answer]
<p>Probably it takes more than a day to change the gh-page. Try to check if the options for that page still right, you can find them in settings-&gt;options and near the bottom before the danger zone there is “Github Pages”. Maybe the source changed.</p>
<p>-Gabriele-</p>

**Comment by PeterGyger** [comment]
<p>Hello</p>
<p>Thank you for your Reply.</p>
<p>GitHub write it needs maximum 10 Min and not one Day.</p>
<p>In these case i never change this TLD Setting. </p>
<p>Sorry for the leazy Question: What are the Alternatives to use GitHub with a TLD like <a href="http://mydomain.org" rel="nofollow noopener">mydomain.org</a> or <a href="http://mydomain.net" rel="nofollow noopener">mydomain.net</a>?</p>
<p>Greetings</p>
<p>Peter</p>

**Comment by Haritsinh** [comment]
<p>Hey <a class="mention" href="/petergyger">@petergyger</a> , if you are looking for how to set custom domain name like (<a href="http://www.test.com" rel="nofollow noopener">www.test.com</a>) on github pages then here is the info page you shuld take a look <a href="https://help.github.com/en/articles/using-a-custom-domain-with-github-pages" rel="nofollow noopener">https://help.github.com/en/articles/using-a-custom-domain-with-github-pages</a> , i hope it helps you.</p>
